decidim-core/app/views/decidim/processes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<main class="wrapper">
+  <% if hero_processes.any? %>
+    <section id ="highlighted-processes" class="row section">
+      <h1 class="section-heading"><%= t(".hero_processes", scope: "layouts") %></h1>
+        <% hero_processes.each do |hero_process| %>
+          <%= render partial: "hero_process", locals: { hero_process: hero_process } %>
+        <% end %>
+    </section>
+  <% end %>
+
+  <section id="processes-grid" class="section row collapse">
+    <%= render partial: "order_by_processes" %>
+    <div class="row small-up-1 medium-up-2 large-up-3 card-grid">
+      <% participatory_processes.each do |participatory_process| %>
+        <%= render partial: "process", locals: { participatory_process: participatory_process } %>
```

Comment:
You can do:

`<%= render processes || "Text to show when there are no processes" %>` and Rails will know which partial to use. In the case of hero_processes, you can do this:

`<%= render hero_processes, partial: "hero_process", as: :hero_process %>`